addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/MongoDbInjectionScanRule.java

```diff
@@ -374,72 +355,77 @@ public void scan(HttpMessage msg, String param, String value) {
             String phase = null;
             boolean hadTimeout = false;
             String sleepValueToInj;
+            String sleepPayload = SLEEP_INJECTION[index][0];
             while (index < SLEEP_INJECTION.length) {
                 if (isStop()) {
                     LOGGER.debug(STOP_LOG);
                     return;
                 }
-                LOGGER.debug("Trying  with the value: {}", SLEEP_INJECTION[index][0]);
+                AtomicReference<HttpMessage> message = new AtomicReference<>();
+                paramValue = sleepPayload.replace("{0}", String.valueOf(timeSleepSeconds));
+                LOGGER.debug("Trying  with the value: {}", sleepPayload);
```

Comment:
I know the problem was there before but the double space may as well be fixed :wink: